toolkits/springernature/packages/springernature-header/HISTORY.md

```diff
@@ -0,0 +1,4 @@
+# History
+
+## 1.0.0 (2020-10-21)
```

Comment:
Maybe start with version `0.1.0` as this indicates that this is new and possibly subject to change. Tend to only go to version `1.0.0` when it is stable and in use across multiple projects